install_config/redeploying_certificates.adoc

```diff
@@ -563,26 +563,48 @@ Then, run the following to remove the environment variables:
 $ oc env dc/router OPENSHIFT_CA_DATA- OPENSHIFT_CERT_DATA- OPENSHIFT_KEY_DATA- OPENSHIFT_MASTER-
 ----
 
-. Delete the `router-certs` secret:
+. To obtain a new certificate, run:
 +
 ----
-$ oc delete secret router-certs
+# cd /root
+# mkdir cert ; cd cert
+# CA=/etc/origin/master
+# oadm ca create-server-cert --signer-cert=$CA/ca.crt --signer-key=$CA/ca.key --signer-serial=$CA/ca.serial.txt --hostnames=hostnames.for.the.certificate --cert=router.crt --key=router.key
+----
++
+This will generate:
++
+- A new certificate (*_router.crt_* in this example).
+- Its corresponding private key (*_router.key_* in this example).
+- A copy of the signing certificate authority (CA) certificate chain (*_$CA/ca.crt_* in this example; it can contain more than one certificate if intermediate CAs are involved).
+
+. Create a new file by concatenating these three files in that specific order:
++
+----
+# cat router.crt router.key $CA/ca.crt > router.pem
 ----
 
-. Remove the following annotations from the `router` service:
+. To back up the old certificate:
 +
 ----
-$ oc annotate service router \
-    service.alpha.openshift.io/serving-cert-secret-name- \
-    service.alpha.openshift.io/serving-cert-signed-by-
+# oc export secret router-certs > ~/old-router-certs-secret.yaml  <1>
+----
+<1> `router-certs` is the default name of the secret, as it is the name used by  the `oadm router --default-cert` option.
```

Comment:
Just one tiny thing, there's a double space between "by" and "the". Otherwise, LGTM! 🌟